### PR TITLE
MLE:12688 Added logic to add enableCustomTranslation for safari>11 and fixed window handles

### PR DESCRIPTION
**Pull request by shiva-guntoju**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,7858 +0,0 @@
-{
-  "name": "testcafe-browser-provider-lambdatest",
```
Comment (Shivanshu-lambdatest) [inline package-lock.json:2]: This should also be included in the repository.

**Review by Shivanshu-lambdatest — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (shiva-guntoju) [reply package-lock.json:2]: Included package-lock.json, please check now

**Review by Shivanshu-lambdatest — APPROVED** [review]
LGTM

**Review by Shivanshu-lambdatest — APPROVED** [review]

**Comment by Shivanshu-lambdatest** [comment]
upgrade version in package.json and also include package lock in PR

**Pull request merged**